backend/src/fun_forecast_backend/weather/weather.py

```diff
@@ -9,11 +9,30 @@
 
 from fun_forecast_backend.core.standard_logger import get_logger
 from fun_forecast_backend.shared.dataclasses import WeatherCondition, HourData
-from fun_forecast_backend.weather.dataclasses import WeatherApiPoint, WeatherData, Period
+from fun_forecast_backend.weather.dataclasses import WeatherApiPoint, WeatherData, Period, LatLongResponse
 
 logger = get_logger()
 
 
+def get_location_by_lat_and_long(lat: float, long: float) -> LatLongResponse:
+    try:
+        point = point_from_coords(lat, long)
+        city = point.properties.relativeLocation.properties.city.strip()
```

Comment:
Can we assume that every coord is going to have a `city` value? Worth testing, not sure the docs are really worth looking at 💩 but maybe

Reply:
Hmmm good point. Let me see what I can dig up